pages/common/age.md

```diff
@@ -1,7 +1,7 @@
 # age
 
 > A simple, modern and secure file encryption tool.
-> See `age-keygen` for how to generate key pairs.
+> See also: `age-keygen`, for generating key pairs.
```

Comment:
What's wrong with the old way of saying it? In my opinion, using "See also" everywhere makes it less readable, **especially** when there are multiple, semicolon-separated pages being referenced.

I know this was [standardized in the style guide](https://github.com/tldr-pages/tldr/pull/11868), but I still want to discuss it.

How about using "See also: `command1`, `command2`" when there is no additional description, and "See `command` for ..." otherwise?

Reply:
> How about using "See also: `command1`, `command2`" when there is no additional description, and "See `command` for ..." otherwise?

Would be a good solution imo

Reply:
```suggestion
> See also: `age-keygen` for generating key pairs.
```